src/components/StockForBuying.vue

```diff
@@ -1,26 +1,25 @@
 <template>
-  <el-col :md="12" :lg="8" :xl="6">
-    <el-card class="card">
-      <div slot="header">
-        <h3 class="card__header">{{ stock.name }}</h3>
-        (<small>Price: {{ stock.price }}</small>)
-      </div>
-      <div class="card__body">
-        <el-input
-          type="number"
+  <el-card class="card">
+    <div slot="header">
+      <h3 class="card__header">{{ stock.name }}</h3>
+      (<small>Price: {{ stock.price }}</small>)
+    </div>
+    <div class="card__body">
+        <el-input-number
```

Comment:
Use correct indentation: 2 instead 4 spaces. - please validate entire project on this